src/cmd/linuxkit/pkglib/manifest_push_script.go

```diff
@@ -48,7 +48,7 @@ esac
 # Push manifest list
 OUT=$(manifest-tool $MT_ARGS push from-args \
                     --ignore-missing \
-                    --platforms linux/amd64,linux/arm64 \
+                    --platforms linux/amd64,linux/arm64,linux/s390 \
```

Comment:
and here?

Reply:
Is it really taste and not what are the `manifest-tool` requirements? (or perhaps it is hub, or the eventual Docker engine which will pull the image, or OCI requirements?)

https://github.com/estesp/manifest-tool seems to call it `s390x` for example.

Reply:
ok, you are right, it depends on the tag. Before multi-arch, the docker images for z platforms were tagged with `s390x`. Anyway, I already pushed with `s390x`

Reply:
I think notionally we support 31 bit s390 images running on s390x, although I don't think anyone has ever tried running one...